.github/workflows/prerelease.yml

```diff
@@ -177,8 +182,8 @@ jobs:
       - name: Generate the results
         run: |
           python3 llvm-test-suite/utils/vast_compare.py \
-            --columns vast-hl,vast-llvm,vast-bin \
-            --files results-hl.json,results-llvm.json,results-bin.json \
+            --columns HighLevel,"HighLevel without Unsupported",LLVM,Binary \
+            --files results-hl-OFF.json,results-hl-ON.json,results-llvm-OFF.json,results-bin-OFF.json \
```

Comment:
use better name than `-OFF` and `-ON`. Something like `results-hl-with-unsup`, `results-hl`.